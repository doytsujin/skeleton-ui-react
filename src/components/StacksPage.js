import React from 'react';

import Stack from './Stack';

const StacksPage = (props) => (
  <div className="page">
    <div className="container">
      <h2 className="display-4">Technology Stacks</h2>
      <div className="row">
        <div className="col-lg col-md-6 col-sm-12 mb-3">
          <Stack name="Application" type="application" />
        </div>
        <div className="col-lg col-md-6 col-sm-12 mb-3">
          <Stack name="Test" type="test" />
        </div>
        <div className="col-lg col-md-6 col-sm-12 mb-3">
          <Stack name="Build" type="build" />
        </div>
      </div>
    </div>
  </div>
);


export default StacksPage;