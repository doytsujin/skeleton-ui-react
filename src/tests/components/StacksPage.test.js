import React from 'react';
import { shallow } from 'enzyme';

import StacksPage from '../../components/StacksPage';

test('should render StacksPage correctly', () => {
  const wrapper =  shallow(<StacksPage />);
  expect(wrapper).toMatchSnapshot();
});