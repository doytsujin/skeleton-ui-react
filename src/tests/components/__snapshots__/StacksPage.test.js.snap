// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StacksPage correctly 1`] = `
<div
  className="page"
>
  <div
    className="container"
  >
    <h2
      className="display-4"
    >
      Technology Stacks
    </h2>
    <div
      className="row"
    >
      <div
        className="col-lg col-md-6 col-sm-12 mb-3"
      >
        <ConnectFunction
          name="Application"
          type="application"
        />
      </div>
      <div
        className="col-lg col-md-6 col-sm-12 mb-3"
      >
        <ConnectFunction
          name="Test"
          type="test"
        />
      </div>
      <div
        className="col-lg col-md-6 col-sm-12 mb-3"
      >
        <ConnectFunction
          name="Build"
          type="build"
        />
      </div>
    </div>
  </div>
</div>
`;
